bedrock/settings/static_media.py

```diff
@@ -1700,6 +1706,12 @@
         ),
         'output_filename': 'js/plugincheck-bundle.js',
     },
+    'plugincheck-update': {
+        'source_filenames': (
+            'js/mozorg/plugincheck-update.js',
```

Comment:
Nit: suggest moving this file to `js/plugincheck/` for consistency as long as the other files are still around?